src/renderer/components/Index/Inventory.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+.backgrounds_container
+    button.hover-button
+        p Inventory
+    - for (let a=0; a < 10; a++)
+        .invetory-item
+</template>
+
+<style lang="stylus">
```

Comment:
Use `<style lang="stylus" scoped>` instead of `<style lang="stylus">`. Scoped means that this css only applies to this file.